python/spear/game_world_service.py

```diff
@@ -2,9 +2,92 @@
 # Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
 #
 
+from enum import Enum
 import json
 
 class GameWorldService():
+
+    class PropertyDesc():
+        property_
+        value_ptr_
+
+    # should match the EIncludeSuperFlag::Type enum in Engine\Source\Runtime\CoreUObject\Public\UObject\Class.h
+    class EIncludeSuperFlag(Enum):
+        ExcludeSuperFlag = 0
+        IncludeSuperFlag = 1
+
+    # should match the ELoadFlags class in Engine\Source\Runtime\CoreUObject\Public\UObject\ObjectMacros.h
```

Comment:
no need to repeat all of these comments, since we're already linking to the Unreal source file

Reply:
See my most recent design sketch to avoid duplicating most of these enums in Python.